test/validate-reviews.test.js

```diff
@@ -0,0 +1,18 @@
+import app from "../server/server";
+import { describe, expect, test } from "@jest/globals";
+import validateReviews from "../server/validateReviews";                                                           
+
+describe("Test review validation", () => {
+    test("rating should be integer between 0 and 5", async () => {
+        const review = {
+            
+            author: 'kalle',
+            rating: 3,
+            comment: "medelmåttigt"
+        }
+    })
```

Comment:
These lines make up the test. The only thing that they do is create an object called `review`, but it doesn't do anything with it.